DMX.Portal.Web/Startup.cs

```diff
@@ -66,6 +67,7 @@ private static void AddServices(IServiceCollection services)
         {
             services.AddTransient<ILabService, LabService>();
             services.AddTransient<ILabViewService, LabViewService>();
+            services.AddSyncfusionBlazor();
```

Comment:
Think this could be moved outside the `AddServices()` method to where the other OOB registrations are